server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -514,7 +514,9 @@ public void run() {
     }
 
     // need to regularly fetch data so plot data is updated
-    Threads.createThread("Data fetcher", () -> {
+    // TODO KEVIN RATHBUN don't think this is a critical function of the Monitor (and the
+    // RuntimeException is already handled here)
+    Threads.createNonCriticalThread("Data fetcher", () -> {
```

Comment:
don't think this is a critical function of the Monitor (and the RuntimeException is already handled here)

Reply:
Even though it handles any exception, would be good to mark this as critical.  If this thread stops running some data displayed on the monitor would be be frozen and would never update.

Reply:
Addressed in 2c4ce6bc57b166a2a57eb61c11df5b2bb8da76b8